nx.json

```diff
@@ -14,7 +13,7 @@
     }
   },
   "workspaceLayout": {
-    "appsDir": "e2e",
+    "appsDir": "apps",
```

Comment:
The reason why I put `e2e` to a separate folder is because this repo is package based, and it is a mirror of NX GitHub repo, which uses the same approach for e2e tests. Now does having an `apps` folder, does it make it integrated repo?

Reply:
As far as I know, this precise option of defining the workspace layout is deprecated, now in Nx you can place projects whereever you like. I thought of just removing it completely but perhaps that would require of upgrade of Nx...

I can try that out and see if you like it more! 😊

Reply:
Yes, I think `workspaceLayout` can be removed, it's not needed anymore. But I'm wondering if your new project would be best to be added to `packages` folder, as at the end it is published as a package?

Reply:
I have moved it all the way back and the code to `packages`, that way we don't have to do extra changes